src/encoder/encoderfdkaac.h

```diff
@@ -205,10 +206,10 @@ class EncoderFdkAac : public Encoder {
     EncoderCallback* m_pCallback;
     std::unique_ptr<QLibrary> m_pLibrary;
     FIFO<SAMPLE>* m_pInputFifo;
-    SAMPLE* m_pFifoChunkBuffer;
+    std::vector<SAMPLE> m_pFifoChunkBuffer;
```

Comment:
The `p` prefix of those member names is now misleading, because we are no longer dealing with (plain) pointers here. That's why I prefer not to add any type prefixes as we still require in our coding style, because they soon become confusing and ambiguous. They are not appropriate for modern C++ code.